src/hooks/useEditorView.ts

```diff
@@ -137,5 +146,13 @@ export function useEditorView<T extends HTMLElement = HTMLElement>(
     if (stateProp) view?.setProps({ state: stateProp });
   }, [view, stateProp]);
 
+  useLayoutEffect(() => {
+    editorPropsRef.current = withConditionalFlushDispatch(
+      props,
+      forceUpdate,
+      view
+    );
+  }, [props, view]);
```

Comment:
Does this layout effect need to be declared before the one where we update the view? As it is, I believe the view will be updated with the _old_ props on a props change, because the ref won't have been updated yet when that effect runs

Reply:
That sounds right to me 👍